lib/parallel_tests/cli.rb

```diff
@@ -192,7 +192,16 @@ def execute_shell_command_in_parallel(command, num_processes, options)
     def report_time_taken
       start = Time.now
       yield
-      puts "\nTook #{Time.now - start} seconds"
+      elapsed = Time.now - start
+      display_elapsed = display_duration(elapsed)
+      puts "\nTook #{elapsed} seconds" + (display_elapsed.include?(':') ? " (#{display_elapsed})" : "")
+    end
+
+    def display_duration(elapsed)
+      rounded = elapsed.round
+      display_parts = [ rounded / 3600, rounded % 3600 / 60, rounded % 60 ].drop_while(&:zero?)
+      display_parts = [0] if display_parts.empty?
```

Comment:
Personally I prefer to see `6:23` rather than `00:06:23`

The workaround for < 0.5s duration was in there just to satisfy the test; take it out if you like.